keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+  * |   1  |   2  |　 3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   !  |   "  |   #  |   $  |   %  |   &  |   '  |  (　 |   )  |   _  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   =  |   ¥  |  {   |   }  |   *  |   ~  |   |  |  <   |   >  |   ?  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  |  Alt |      |   [  |   ]  |   ;  |   :  |      |   \  |   /  |
+  * `---------------------------------------------------------------------'
+  */
+ [_RAISE] = LAYOUT( \
+   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, \
+   KC_EXLM, KC_DQT,  KC_HASH, KC_DLR,  KC_PERC, KC_AMPR, KC_QUOT, KC_LPRN, KC_RPRN, KC_UNDS,  \
+   KC_EQL,  KC_JYEN, KC_LCBR, KC_RCBR, KC_ASTR, KC_TILD, KC_PIPE, KC_LABK, KC_RABK, KC_QUES,  \
+   KC_F1,   KC_LALT, _______, KC_LBRC, KC_RBRC, KC_SCLN, KC_COLN, _______, KC_BSLS, KC_SLSH   \
+ ),
+
+ /* EUCALYN
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   ,  |   .  |   ;  |   M  |   R  |   D  |   Y  |   P  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   O  |   E  |   I  |   U  |   G  |   T  |   K  |   S  |   N  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   F  |   B  |   H  |   J  |   L  |   /  |
+  * `-------------+------+------+------+------+------+------+-------------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_EUCALYN] = LAYOUT( \
+   KC_Q,     KC_W,      KC_COMM,   KC_DOT,    KC_SCLN,     KC_M,     KC_R,     KC_D,     KC_Y,      KC_P, \
+   KC_A,     KC_O,      KC_E,      KC_I,      KC_U,        KC_G,     KC_T,     KC_K,     KC_S,      KC_N, \
+   KC_Z,     KC_X,      KC_C,      KC_V,      KC_F,        KC_B,     KC_H,     KC_J,     KC_L,      KC_SLSH, \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Adjust (Lower + Raise)
+  * ,---------------------------------------------------------------------.
+  * |      |RGB Tg|RGB Md|Hue Up|Hue Dn|Sat Up|Sat Dn|Val Up|Val Dn|      |
+  * |------+------+------+------+------+------.------+------+------+------|
+  * |      |      |      |Aud on|Audoff|AGnorm|AGswap|      |BL TOG|BL STP|
+  * |------+------+------+------+------+------+------+------+------+------|
+  * |Qwerty|Colemk|Dvorak|Eucalyn|     |      |      |      |      |      |
+  * |------+------+------+------+------+------+------+------+------+------|
+  * |      |      |      |      |      |      |      |      |      |      |
+  * `---------------------------------------------------------------------'
+  */
+ [_ADJUST] =  LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_ADJUST] =  LAYOUT_ortho_4x10( \
```